packages/big-design/src/components/Table/SelectAll/SelectAll.tsx

```diff
@@ -54,6 +54,9 @@ export const SelectAll = <T extends TableItem>({
 
   const label = allInPageSelected ? 'Deselect All' : 'Select All';
 
+  const nonSelectionSummary = totalItems || '';
```

Comment:
If `totalItems` is 0, wouldn't this be `false`, as in it'll show up in the UI as `false`? Should we use null coalescing instead?
```suggestion
  const nonSelectionSummary = totalItems ?? '';
```

I think there is another place that does this too.

Reply:
fixed: https://github.com/bigcommerce/big-design/pull/1602/commits/81e592f6d0fc61ff3311a757aa44723d4c9c16c2